CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ option(USE_SDL "Use SDL" OFF)
 option(USE_RLGLUE "Use RL-Glue" OFF)
 option(BUILD_EXAMPLES "Build Example Agents" ON)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wunused -fPIC -O3 -fomit-frame-pointer")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wunused -Wno-multichar -fPIC -O3 -fomit-frame-pointer -D__STDC_CONSTANT_MACROS")
```

Comment:
Regarding the other changes, I am submitting the fix now.

On 1 June 2015 at 16:48, Marlos C. Machado <marlos.cholodovskis@gmail.com>
wrote:

> I did that, because of the warnings I was getting in the Linux.
> I wrote that on the PR, with more details, if you want to check.
>
> On 1 June 2015 at 16:42, Marc G. Bellemare <notifications@github.com>
> wrote:
>
>> In CMakeLists.txt
>> <https://github.com/mgbellemare/Arcade-Learning-Environment/pull/59#discussion_r31478414>
>> :
>>
>> > @@ -5,7 +5,7 @@ option(USE_SDL "Use SDL" OFF)
>> >  option(USE_RLGLUE "Use RL-Glue" OFF)
>> >  option(BUILD_EXAMPLES "Build Example Agents" ON)
>> >
>> > -set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wunused -fPIC -O3 -fomit-frame-pointer")
>> > +set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wunused -Wno-multichar -fPIC -O3 -fomit-frame-pointer -D__STDC_CONSTANT_MACROS")
>>
>> The -Wno-multichar keeps coming back. Is this on purpose?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/mgbellemare/Arcade-Learning-Environment/pull/59/files#r31478414>
>> .
>>
>
>
>
> --
> Marlos Cholodovskis Machado
> http://www.dcc.ufmg.br/~marlos <http://www.cs.ualberta.ca/~machado>
>



-- 
Marlos Cholodovskis Machado
http://www.dcc.ufmg.br/~marlos <http://www.cs.ualberta.ca/~machado>